recipes/mp-units/config.yml

```diff
@@ -3,3 +3,5 @@ versions:
     folder: 0.8.0
   "2.0.0":
     folder: 2.0.0
+  "2.1.0":
+    folder: 2.0.0
```

Comment:
If you could order the newer versions at the top that would be great.

Reply:
Done